composition/interfaces.go

```diff
@@ -83,7 +83,7 @@ type Content interface {
 	Tail() Fragment
 
 	// The attributes for the body element
-	BodyAttributes() Fragment
+	BodyAttributes() []html.Attribute
```

Comment:
Breaking API change, please leave the old method and create BodyAttributesV2 or something